src/main/java/net/minecraftforge/common/IPlantable.java

```diff
@@ -29,18 +29,7 @@
 
 public interface IPlantable
 {
-    default PlantType getPlantType(BlockGetter world, BlockPos pos) {
-        if (this instanceof CropBlock) return PlantType.CROP;
-        if (this instanceof SaplingBlock) return PlantType.PLAINS;
-        if (this instanceof FlowerBlock) return PlantType.PLAINS;
-        if (this == Blocks.DEAD_BUSH)      return PlantType.DESERT;
-        if (this == Blocks.LILY_PAD)       return PlantType.WATER;
-        if (this == Blocks.RED_MUSHROOM)   return PlantType.CAVE;
-        if (this == Blocks.BROWN_MUSHROOM) return PlantType.CAVE;
-        if (this == Blocks.NETHER_WART)    return PlantType.NETHER;
-        if (this == Blocks.TALL_GRASS)      return PlantType.PLAINS;
-        return net.minecraftforge.common.PlantType.PLAINS;
-    }
+    PlantType getPlantType(BlockGetter world, BlockPos pos);
```

Comment:
Is there a reason instead of having a default here like there was previously, just with updated values? That would greatly reduce the maintenance overhead of the system.